bin/test_optional_dependencies.py

```diff
@@ -85,6 +85,8 @@ class TestsFailedError(Exception):
 
 doctest_list = [
     # numpy
+    'doc/src/tutorials/biomechanics/biomechanical-model-example.rst',
+    'doc/src/tutorials/biomechanics/biomechanics.rst',
```

Comment:
Probably all tutorials should be tested here.

I hadn't really noticed before that this section does not list anything from the rst docs so maybe more things should be added.